browsers/browsers.json

```diff
@@ -0,0 +1,1223 @@
+        "status": "retired"
+      },
+      "9": {
+        "release_date": "2011-03-14",
+        "status": "retired"
+      },
+      "10": {
+        "release_date": "2012-10-26",
+        "status": "retired"
+      },
+      "11": {
+        "release_date": "2013-10-17",
+        "status": "current"
+      }
+    }
+  },
+  "opera": {
+    "releases": {
+      "2": {
+        "release_date": "1996-12-01",
+        "status": "retired"
+      },
+      "3": {
+        "release_date": "1997-12-01",
+        "status": "retired"
+      },
+      "3.5": {
+        "release_date": "1998-11-18",
+        "status": "retired"
+      },
+      "3.6": {
+        "release_date": "1999-05-06",
+        "status": "retired"
+      },
+      "4": {
+        "release_date": "2000-06-28",
+        "status": "retired"
+      },
+      "5": {
+        "release_date": "2000-12-06",
+        "status": "retired"
+      },
+      "5.1": {
+        "release_date": "2001-04-10",
+        "status": "retired"
+      },
+      "6": {
+        "release_date": "2001-12-18",
+        "status": "retired"
+      },
+      "7": {
+        "release_date": "2003-01-28",
+        "status": "retired"
+      },
+      "7.1": {
+        "release_date": "2003-04-11",
+        "status": "retired"
+      },
+      "7.2": {
+        "release_date": "2003-09-23",
+        "status": "retired"
+      },
+      "7.5": {
+        "release_date": "2004-05-12",
+        "status": "retired"
+      },
+      "8": {
+        "release_date": "2005-04-19",
+        "status": "retired"
+      },
+      "8.5": {
+        "release_date": "2005-09-20",
+        "status": "retired"
+      },
+      "9": {
+        "release_date": "2006-06-20",
+        "status": "retired"
+      },
+      "9.1": {
+        "release_date": "2006-12-18",
+        "status": "retired"
+      },
+      "9.2": {
+        "release_date": "2007-04-11",
+        "status": "retired"
+      },
+      "9.5": {
+        "release_date": "2008-06-12",
+        "status": "retired"
+      },
+      "9.6": {
+        "release_date": "2008-10-08",
+        "status": "retired"
+      },
+      "10": {
+        "release_date": "2009-09-01",
+        "status": "retired"
+      },
+      "10.1": {
+        "release_date": "2009-11-23",
+        "status": "retired"
+      },
+      "10.5": {
+        "release_date": "2010-03-02",
+        "status": "retired"
+      },
+      "10.6": {
+        "release_date": "2010-07-01",
+        "status": "retired"
+      },
+      "11": {
+        "release_date": "2010-12-16",
+        "status": "retired"
+      },
+      "11.1": {
+        "release_date": "2011-04-12",
+        "status": "retired"
+      },
+      "11.5": {
+        "release_date": "2011-06-28",
+        "status": "retired"
+      },
+      "11.6": {
+        "release_date": "2011-12-06",
+        "status": "retired"
+      },
+      "12": {
+        "release_date": "2012-06-14",
+        "status": "retired"
+      },
+      "12.1": {
```

Comment:
2012-11-05 ?

Reply:
2012-11-06 according to http://www.opera.com/docs/history/presto/#o1210

Reply:
FIne then.